morphia/src/main/java/org/mongodb/morphia/Morphia.java

```diff
@@ -255,7 +266,7 @@ public synchronized Morphia mapPackage(final String packageName, final boolean i
      * @return this
      */
     public Morphia mapPackageFromClass(final Class clazz) {
-        return mapPackage(clazz.getPackage().getName(), false);
+        return mapPackage(clazz.getPackage().getName(), false, true);
```

Comment:
I agree with the default of ignoring subpackages to avoid an invisible change of functionality but this feels a bit asymmetric.  I think the option to map subpackages should be on MapperOptions rather than adding another opaque method parameter.